RELEASE_NOTES.md

```diff
@@ -1,3 +1,9 @@
+# 0.2.0
```

Comment:
None of these change the service behavior so I think the version can just be 0.1.3. Also the version needs to be bumped in the service code, see https://github.com/kbase/blobstore?tab=readme-ov-file#developers

Reply:
👍